tsconfig.json

```diff
@@ -2,7 +2,11 @@
   "extends": "./tsconfig.paths.json",
   "compilerOptions": {
     "target": "es2019",
-    "lib": ["dom", "dom.iterable", "es2019"],
```

Comment:
I see this formatting keeps happening when running the react-scripts, does adding the tsconfig to .prettierignore fixes that?

Reply:
Yeah, the problem is I think that `react-scripts` enforces specific formatting but Prettier tries to enforce our styleguide so we go back and forth, that's why I added it to `.prettierignore` and we're just gonna leave it be